twilight-util/src/builder/embed/mod.rs

```diff
@@ -122,8 +122,7 @@ impl EmbedBuilder {
 
     /// Set the color.
     ///
-    /// This must be a valid hexadecimal RGB value. `0x000000` is not an
-    /// acceptable value as it would be thrown out by Discord. Refer to
+    /// This must be a valid hexadecimal RGB value. Refer to
```

Comment:
Maybe also add the ignore note here:
```suggestion
    /// This must be a valid hexadecimal RGB value. `0x000000` is ignored and
    /// doesn't count towards the final computed color in the user list. Refer to
```

Reply:
is this true?

Reply:
Ah no, I just confused sth. for a sec.